visual-retrieval-colpali/frontend/app.py

```diff
@@ -549,32 +560,45 @@ def SearchResult(results: list, query_id: Optional[str] = None):
             ),
         )
 
-    return Div(
-        *result_items,
-        image_swapping,
-        toggle_text_content,
-        dynamic_elements_scrollbars,
-        id="search-results",
-        cls="grid grid-cols-1 gap-px bg-border min-h-0",
-    )
+    return [
+        Div(
+            *result_items,
+            image_swapping,
+            toggle_text_content,
+            dynamic_elements_scrollbars,
+            id="search-results",
+            cls="grid grid-cols-1 gap-px bg-border min-h-0",
+        ),
+        Div(
+            ChatResult(query_id=query_id, query=query, doc_ids=doc_ids),
+            hx_swap_oob="true",
+            id="chat_messages",
+        ),
+    ]
+
 
+def ChatResult(query_id: str, query: str, doc_ids: Optional[list] = None):
+    messages = Div(LoadingSkeleton())
+
+    if doc_ids:
+        messages = Div(
+            LoadingSkeleton(),
+            hx_ext="sse",
+            sse_connect=f"/get-message?query_id={query_id}&doc_ids={",".join(doc_ids)}&query={quote_plus(query)}",
```

Comment:
```suggestion
            sse_connect=f"/get-message?query_id={query_id}&doc_ids={','.join(doc_ids)}&query={quote_plus(query)}",
```
I got this error when running: 
```python
Traceback (most recent call last):
  File "/Users/thomas/Repos/sample-apps/visual-retrieval-colpali/main.py", line 33, in <module>
    from frontend.app import (
  File "/Users/thomas/Repos/sample-apps/visual-retrieval-colpali/frontend/app.py", line 587
    sse_connect=f"/get-message?query_id={query_id}&doc_ids={",".join(doc_ids)}&query={quote_plus(query)}",
                                                             ^
SyntaxError: f-string: expecting '}'
```